test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,22 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
Recall from tutorial if lumping all test conditions within a Junit test or separating into different Junit test based on test condition is a better implementation?